src/base/bittorrent/session.h

```diff
@@ -402,6 +402,9 @@ namespace BitTorrent
         void setAnnounceIP(const QString &ip);
         int maxConcurrentHTTPAnnounces() const;
         void setMaxConcurrentHTTPAnnounces(int value);
+        bool isReannounceWhenAddressChanged() const;
+        void setReannounceWhenAddressChanged(bool enabled);
+        void reannounceToAllTrackers();
```

Comment:
And I would suggest renaming it to `forceReannounceToAllTrackers`

Reply:
I think there is some difference between `force reannounce` and `reannounce without delay` of this PR. 

It would be confusing to someone if `force` is used here.

https://github.com/qbittorrent/qBittorrent/pull/14650#issuecomment-820654847

Reply:
To make the naming consistent with other vars and more explicit to others, I renamed them to `*ReannounceWhenAddressChangedEnabled`.

https://github.com/qbittorrent/qBittorrent/pull/14650#discussion_r635091080